blogposts/2021/release-3.31.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Sourcegraph 3.31 release"
+publishDate: 2021-08-20T10:00-07:00
+description: "Sourcegraph 3.31 introduces UPDATE"
+tags: [blog, release]
+slug: "release/3.31"
+published: false
+heroImage: 
+socialImage: 
+changelogItems:
+  - description: "Backend Code Insights GraphQL queries now support arguments `includeRepoRegex` and `excludeRepoRegex` to filter on repository names."
+    url: https://github.com/sourcegraph/sourcegraph/pull/23256
+    category: Code Insights
+  - description: Operator documentation has been added to the Search Reference sidebar section.
+    url: https://github.com/sourcegraph/sourcegraph/pull/23116
+    category: Search
+  - description: Syntax highlighting is now supported for CUE.
+    url: https://cuelang.org
+    category: Search
+  - description: Reintroduced a revised version of the Search Types sidebar section, making the quick links for search types (for example, diff searches) more intuitive to use.
+    url: https://github.com/sourcegraph/sourcegraph/pull/23170
+    category: Search
+  - description: "Added support for git p4's `--use-client-spec` option, allowing Sourcegraph to avoid cloning unimportant files. This can now be enabled by configuring the `p4.client` field in the Perforce integration."
+    url: https://github.com/sourcegraph/sourcegraph/pull/23845
+    category: Admin
+  - description: "Batch Changes changesets can now be set to published when previewing new or updated batch changes."
+    url: https://docs.sourcegraph.com/batch_changes/how-tos/publishing_changesets#within-the-ui
+    category: Batch Changes
```

Comment:
@malomarrec Is this simply saying that batch changes can now be published from the UI? Would it be accurate to rewrite it that way?